students/tri_nguyen/lesson03/list_lab.py

```diff
@@ -0,0 +1,179 @@
+num_from_user = num_from_user - 1  # Python uses zero based indexing
+
+if fruit_list[num_from_user] in fruit_list[num_from_user]:
+    print(num_from_user + 1, '-', fruit_list[num_from_user])
+
+print()
+fruit_list = ['Watermelons'] + fruit_list
+print('Watermelons has been added to the beginning of the list.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+fruit_list.insert(0, 'Pinnaples')
+print('Pinnaples have been added to the beginning of the list.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+print('Below is the list of fruits that begin with "P".')
+for fruit in fruit_list:
+    if fruit.startswith('P'):
+        print(fruit)
+
+# End of Series 1
+# ------------------------------------------- #
+
+# ------------------------------------------- #
+# Series 2 - Using the list created in series 1:
+# - Display the list.
+# - Remove the last fruit from the list.
+# - Display the list.
+# - Ask the user for a fruit to delete, find it
+#   and delete it.
+# - (Bonus: Multiply the list times two. Keep asking
+#    until a match is found. Once found, delete all
+#    occurrences).
+
+print('\n'+'--Series 2 of the exercise--'.upper()+'\n')
+
+print('Below is the current list of fruits.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+print('The last fruit ({}) from the list has been removed.'.format(fruit_list[len(fruit_list) - 1]))
+fruit_list.pop()
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+to_be_deleted = input('Enter a fruit to delete: ')
+
+if to_be_deleted.title() in fruit_list:
+    fruit_list.remove(to_be_deleted.title())
+
+print('\nBelow is a newly updated list of fruits.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+
+# End of Series 2
+# ------------------------------------------- #
+
+# ------------------------------------------- #
+# Series 3 - Using the list from series 1:
+# - Ask the user for input displaying a line like
+#   'Do you like apples?' for each fruit in the list.
+#   make the fruit all lowercase.
+# - For each 'no', delete that fruit from the list.
+# - For any answer that is not 'yes' or 'no', prompt
+#   the user to answer with one of those two values.
+# - Display the list.
+
+print('\n'+'--Series 3 of the exercise--'.upper()+'\n')
+
+count = 0
+answer_set = ('yes', 'no')
+
+while count <= len(fruit_list):
+
+    for idx, fruit in enumerate(fruit_list):
+
+        from_user = input('Do you like ' + fruit.lower() + '? ')
+
+        if from_user.lower() == answer_set[1]:
+            print("\nYou don't like " + fruit.lower() + ". It'll be removed.\n")
+            fruit_list.remove(fruit)
+        elif from_user.lower() == answer_set[0]:
+            print('\n' + fruit + ' are good.\n')
+        else:
+            print('\nPlease enter yes or no\n')
+            continue
+        count = count + 1
```

Comment:
Regarding your PR comment about a way to make this cleaner.

------
I would choose a `for` loop for main outer loop because you just need to loop through each fruit and ask the user. A `while` loop is a good option to keep prompting user if they didn't answer with yes or no.

Here's an alternate version of what this could look like:
```
fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']

new_fruit_list = []

for fruit in fruit_list:
    like_answer = input(f"Do you like {fruit.lower()}? ")
    
    while like_answer.lower() not in ("yes", "no"):
        like_answer = input("Please answer 'yes' or 'no' ")

    if like_answer == "yes":
        new_fruit_list.append(fruit)

print(new_fruit_list)

```